### DISCUSSION TITLE
Boot Error

### DISCUSSION THREAD

**Discussion by steveo314** [mocaccinoOS/mocaccino#140] [Q&A] [answered]
![Screenshot 2024-09-17 121544](https://github.com/user-attachments/assets/7dfee879-04ae-43be-88aa-e3e463321d66)

I do not know if I've ever seen this error in almost 20 years. All i've done was switch to the non lts kernel. Would i be able to fix this via chroot from the live media?

**Comment by joostruis** [comment, marked as answer]
Hi there!

You might bumped into this issue: https://github.com/mocaccinoOS/mocaccino/issues/109

I would try to chroot in and then manually run: 
`luet install kernel/mocaccino-initramfs`

Then use this command to regenerate the initramfs:
`mocaccino-dracut --rebuild-all`

**Reply by steveo314** [reply]
Thank you for your help. That worked.
